### DISCUSSION TITLE
Cannot use --turbo and cannot upgrade to v15 from v14

### DISCUSSION THREAD

**Discussion by razb-viola** [vercel/next.js#72065] [Help]
### Summary

I am trying to speed up the dev server which is extremely slow. 
Using the --turbo flag in the next dev cause issues and won't let the project get up and running - for example: packages that I am using as a local NPM linked (`npm link`) packages are just not found.

Then I tried to upgrade to the new v15, but the upgrading process get tons of errors - none of them are human readable) - therefore I don't list them here.

**Comment by Grenish** [comment]
To upgrade to next.js 15 you could try using the following commands
```bash
# Use the new automated upgrade CLI
npx @next/codemod@canary upgrade latest
 
# ...or upgrade manually
npm install next@latest react@rc react-dom@rc
```

But as next.js suggests, to upgrade smoothly you should use
```bash
npx @next/codemod@canary upgrade latest
```

After that you could configure `package.json` start command to use `--turbo`
